docker-compose.yml

```diff
@@ -1,5 +1,15 @@
 version: "3"
 services:
+  prometheus:
+    depends_on:
+      - kt-server
+      - kt-signer
+    image: prom/prometheus
+    volumes: 
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./prometheus.rules:/etc/prometheus/prometheus.rules
```

Comment:
Can't we just `ADD`/`COPY` these files in a seperate Dockerfile? This will save us some struggle when running this from kubernetes/gce.